plugins/mumble_plugin_utils.h

```diff
@@ -118,6 +119,37 @@ static inline std::string readFile(const std::string &path) {
 	return content;
 }
 
+/// Reads characters from the stream until \p character is encountered.
+/// @return Read characters and whether the stream is still good.
+static inline std::pair< std::string, bool > readUntil(std::stringstream &stream,
+													   const std::stringstream::int_type character) {
+	std::string buf;
+
+	for (auto input = stream.get(); input != character; input = stream.get()) {
+		if (stream.eof()) {
```

Comment:
Maybe it would make sense to use the implicit bool conversion of a stream object like this:
```suggestion
		if (!stream) {
```
I think that this also covers the EOF state but additionally it also checks `stream.good()` afaik which we probably want to check as well :thinking:

Reply:
https://en.cppreference.com/w/cpp/io/basic_ios/operator_bool